### PR TITLE
WIP: deploy doc updates

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/install/common-prod-deploy-tar.rst [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+    Minimum system requirements include:
+    
+       - Hardware: 2 vCPUs/cores, 4GB RAM, supporting 1,000-2,000 users
+       - Network ports required:
+       
+            - HTTP/websockets ports 8065/80/443, TLS, TCP Inbound
+            - SMTP port 10025, TCP/UDP Outbound
+
+       - Database: MySQL v8+ or PostgreSQL v12+
```
Comment (nab-77) [inline source/install/common-prod-deploy-tar.rst:15]: Move this to right after the hardware item.

File: source/install/common-prod-deploy-tar.rst [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
 These instructions outline how to install Mattermost Server on a 64-bit Linux host from a compressed tarball, and assume the IP address of the Mattermost server is 10.10.10.2.
 
+.. note::
+
+    Minimum system requirements include:
+    
+       - Hardware: 2 vCPUs/cores, 4GB RAM, supporting 1,000-2,000 users
+       - Network ports required:
+       
+            - HTTP/websockets ports 8065/80/443, TLS, TCP Inbound
```
Comment (nab-77) [inline source/install/common-prod-deploy-tar.rst:12]:
```suggestion
            - Application ports 80/443, TLS, TCP Inbound
            - Administrator Console port 8065, TLS, TCP Inbound
```

**Review by nab-77 — APPROVED** [review]

**Review by nab-77 — APPROVED** [review]

File: source/install/common-prod-deploy-omnibus.rst [hunk 3]
```diff
@@ -4,6 +4,19 @@
 
     Mattermost bundles the components of a Mattermost deployment into a single installation, called **Omnibus**. Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. The package bundles the free, unlicensed Mattermost Enterprise version of Mattermost, and leverages the `apt package manager <https://ubuntu.com/server/docs/package-management>`__ to install and update the platform components. A custom CLI and ansible recipes link the components together and configures them.
 
+.. note::
+
+    Minimum system requirements include:
+    
+       - Hardware: 2 vCPUs/cores with 4GB RAM (support for 1,000-2,000 users)
```
Comment (nab-77) [inline source/install/common-prod-deploy-omnibus.rst:11]:
```suggestion
       - Hardware: 2 vCPUs/cores with 4GB RAM (support for 1,000-2,000 users)
       - Database: MySQL v8+ or PostgreSQL v12+
```

File: source/install/common-prod-deploy-tar.rst [hunk 4]
```diff
@@ -2,6 +2,19 @@
 
 These instructions outline how to install Mattermost Server on a 64-bit Linux host from a compressed tarball, and assume the IP address of the Mattermost server is 10.10.10.2.
 
+.. note::
+
+    Minimum system requirements include:
+    
+       - Hardware: 2 vCPUs/cores with 4GB RAM (support for 1,000-2,000 users)
```
Comment (nab-77) [inline source/install/common-prod-deploy-tar.rst:9]:
```suggestion
       - Hardware: 2 vCPUs/cores with 4GB RAM (support for 1,000-2,000 users)
       - Database: MySQL v8+ or PostgreSQL v12+
```

File: source/install/common-prod-deploy-tar.rst [hunk 5]
```diff
@@ -2,6 +2,19 @@
...
+       - Network ports required:
+       
+            - Application ports 80/443, TLS, TCP Inbound
+            - Administrator Console port 8065, TLS, TCP Inbound
+            - SMTP port 10025, TCP/UDP Outbound
+
+       - Database: MySQL v8+ or PostgreSQL v12+
```
Comment (nab-77) [inline source/install/common-prod-deploy-tar.rst:16]:
```suggestion
```

**Review by nab-77 — APPROVED** [review]

File: source/install/common-prod-deploy-tar.rst [hunk 6]
```diff
@@ -2,6 +2,20 @@
...
+    
+       - Hardware: 2 vCPUs/cores with 4GB RAM (support for 1,000-2,000 users)
+       - Database: MySQL v8+ or PostgreSQL v12+
+       - Network ports required:
+       
+            - Application ports 80/443, TLS, TCP Inbound
+            - Administrator Console port 8065, TLS, TCP Inbound
+            - SMTP port 10025, TCP/UDP Outbound
+
+       - Database: MySQL v8+ or PostgreSQL v12+
```
Comment (nab-77) [inline source/install/common-prod-deploy-tar.rst:17]:
```suggestion
```

**Review by nab-77 — APPROVED** [review]

**Review by nab-77 — APPROVED** [review]

**Comment by cwarnermm** [comment]
Proposed updates have been applied via other docs PRs. Closing.

**Pull request closed without merging**